sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/CosmosItemTest.java

```diff
@@ -55,6 +61,7 @@ public class CosmosItemTest extends TestSuiteBase {
         ImplementationBridgeHelpers.CosmosDiagnosticsHelper.getCosmosDiagnosticsAccessor();
 
     private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
+    private static final Random random = new Random();
```

Comment:
Random is not thread-safe (neither in Java nor .Net) - please use ThreadLocalRandom.current instead

Reply:
Not really critical in test code - but there is always the risk that code is copied and pasted - so, better to fix this. Non-blocking comment thought.

Reply:
Fixed in next iteration.